packages/radio-player/src/radio-player.ts

```diff
@@ -716,7 +716,7 @@ export default class RadioPlayer extends LitElement {
     if (detail.volume === undefined) {
       return;
     }
-    this.volume = e.detail.volume;
```

Comment:
You can `git commit --amend` changes to a commit that hasn't been pushed up to origin yet without a force push. Then you'll be able to eliminate the 'bump' commits.

Reply:
Do those commits count as individual commits? I did that because `lerna` seems to count the number of commits to generate the canary version and if you try to re-publish the same version, it fails.